src/main/java/stream/flarebot/flarebot/objects/Report.java

```diff
@@ -17,7 +15,7 @@
     private String reportedId;
     private Timestamp time;
     private ReportStatus status;
-    private List<Message> messages = new CopyOnWriteArrayList<>();
+    private List<ReportMessage> messages = new CopyOnWriteArrayList<>();
```

Comment:
Any reason why you didn't just make a map

Reply:
It would of been harder to implement into the existing systems.

Reply:
Map of what? It's a list of objects, mapping it to a value wouldn't have made much sense.